src/components/AdminPanel/AdminMedCenters/MedCenters/MedCentersList/MedCentersList.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import { fetchMedCenters, medCenterDelete } from '@store/actionCreators/medCenters';
+import { useTypesSelector } from '@hooks/UseTypedSelector';
+import Button from '@components/common/Button/Button';
+import MedCenter from '@components/MedCentersPage/MedCentersList/MedCenter/MedCenter';
+import { MedCentersListWrapper, CenterWrapper } from './MedCentersListStyle';
+import Portal from '@components/common/Portal/Portal';
+import Modal from '@components/common/Modal/Modal';
+import DeleteMedCenterPopup from './DeleteMedCenterPopup.tsx/DeleteMedCenterPopup';
+
+interface IMedCentersListProps {
+  searchValue: string;
+}
+
+const MedCentersList: React.FC<IMedCentersListProps> = ({ searchValue }) => {
+  const dispatch = useDispatch();
+  const { medCenters } = useTypesSelector((state) => state.medCenter);
+  
+  const [modalActive, setModalActive] = useState(false);
+  const [idDeleteCenter, setIdDeletedCenter] = useState<string>('');
+
+  const deleteMedCenter = (idMedCenter: string) => {
+    setModalActive(!modalActive);
+    setIdDeletedCenter(idMedCenter);
```

Comment:
these two callbacks will cause two re-renders

Reply:
I'll leave it like that for a while